src/XMakeTasks/UnitTests/Microsoft.Build.Tasks.UnitTests.csproj

```diff
@@ -185,11 +185,11 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Xaml" />
     <Reference Include="System.Xml" />
-    <Reference Include="PresentationFramework" />
+    <Reference Include="PresentationFramework" Condition="'$(OS)' == 'Windows'" />
```

Comment:
I'd be surprised if the test depends on WPF even on Windows so this can probably just be removed.